morphct/chromophores.py

```diff
@@ -0,0 +1,414 @@
+import copy
+import itertools
+from collections import defaultdict
+import os
+import sys
+from warnings import warn
+
+import ele
+import freud
+from openbabel import openbabel
+from openbabel import pybel
+import numpy as np
+from scipy.spatial import Delaunay
+
+from morphct import execute_qcc as eqcc
+from morphct import helper_functions as hf
+
+
+class Chromophore:
+    """An object for tracking infromation related to the chromophore.
+
+    Chromphore refers to the electronically active part of the molecule where a
+    charge is expected to reside.
+    This class keeps track of what particles from the simulation compose each
+    chromophore, the chromphore's neighbors, and the energy values of each
+    chromophore and chromophore pair.
+
+    Parameters
+    ----------
+    chromo_id : int
+        Index of this chromophore in the system.
+    snap : gsd.hoomd.Snapshot
+        Atomistic simulation snapshot from a GSD file. It is expected that the
+        lengths in this file have been converted to Angstroms.
+    atom_ids : numpy.ndarray of int
+        Snapshot indices of the particles which belong to this chromophore.
+    species : str
+        Chromophore species ('donor' or 'acceptor')
+    conversion_dict : dictionary
+        A dictionary that maps the atom type to its element. e.g., `{'c3': C}`
+        An instance that maps AMBER types to their element can be found in
+        `amber_dict`.
+    reorganization_energy : float, default 0.3064
+        Energy required to "reorganize" the system structure from the final to
+        initial coordinates. (what paper did this default come from? TODO EJ)
+    vrh_delocalization : float, default 2e-10
+        Variable-range hopping (rate? where did default come from?) TODO EJ
```

Comment:
Pretty sure default came from Matty fudging around to find something that gave sensible mobilities.